From d35abc7620c8f06618b4708d9a969dfa2f309e96 Mon Sep 17 00:00:00 2001
From: patrick96 <p.ziegler96@gmail.com>
Date: Wed, 12 Jul 2017 11:22:45 +0200
Subject: [PATCH] Remove curlbuild include

Curl removed that header in 73a2fcea0b4adea6ba342cd7ed1149782c214ae3
([1])
http.cpp doesn't use anything from that header and compilation works for
fine with curl 7.54.1

Fixes #647
Ref:
[1]: https://github.com/curl/curl/commit/73a2fcea0b4adea6ba342cd7ed1149782c214ae3
---
 src/utils/http.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/utils/http.cpp b/src/utils/http.cpp
index 365dd3be..f6fe6779 100644
--- src/utils/http.cpp
+++ src/utils/http.cpp
@@ -1,5 +1,4 @@
 #include <curl/curl.h>
-#include <curl/curlbuild.h>
 #include <curl/easy.h>
 #include <sstream>
 
